core/backend/src/TxnManager.ts

```diff
@@ -31,12 +32,16 @@ export interface ValidationError {
   message?: string;
 }
 
-/**
- * @beta
+/** Describes a set of [[Element]]s or [[Model]]s that changed as part of a transaction.
+ * @see [[TxnManager.onElementsChanged]] and [[TxnManager.onModelsChanged]].
+ * @public
  */
 export interface TxnChangedEntities {
+  /** Ids of entities that were inserted by the transaction. */
   inserted: OrderedId64Array;
+  /** Ids of entities that were deleted by the transaction. */
   deleted: OrderedId64Array;
+  /** Ids of elements that were modified by the transaction. */

```

Comment:
elements -> entities

Do we want to use `modified` in the description and `updated` in the API?  Not necessarily an issue just wasn't sure if there was a preference.

Reply:
I thought using a synonym would help clarify what "updated" means.
Should've been entities, you're right - thanks.